packages/block-library/src/navigation/edit/index.js

```diff
@@ -598,6 +598,7 @@ function Navigation( {
 						) }
 						<h3>{ __( 'Overlay Menu' ) }</h3>
```

Comment:
I wonder what this `h3` with the wonky margin is doing here 😭

Reply:
😬 Maybe not in this PR since it cascades into more than just a line change, but would it make sense to remove the prop `hideLabelFromVision` from the `ToggleGroupControl` and remove the h3? I wonder why we need to hide 'Configure', and if we do, why does it need to be in the label as well? 🤔 Does it add much value in terms of accessibility? 

And another h3 is added for 'Submenus'. Would it make sense to use `BaseControl.VisualLabel` above the `ToggleControl` components? Or is there something else we'd use for just labels?

Reply:
> would it make sense to remove the prop `hideLabelFromVision` from the `ToggleGroupControl` and remove the h3?

I think so! In this case I think it's actually suboptimal for screen reader users since the `h3` and `aria-label` texts are so redundant.

> And another h3 is added for 'Submenus'. Would it make sense to use `BaseControl.VisualLabel` above the `ToggleControl` components?

Sounds good to me. `BaseControl.VisualLabel` is fair game whenever you want something that always _looks_ like the other control labels but is not semantically a control label. It is safer than relying on implicit cascading styles like we have here on all `h3`s in the sidebar 😅